core/server/log.js

```diff
@@ -30,21 +29,10 @@ module.exports = function log(...msg) {
   console.log(d, ...msg)
 }
 
-const throttledConsoleError = throttle(console.error, 10000, {
-  trailing: false,
-})
-
 module.exports.error = function error(...msg) {
   const d = date()
   listeners.forEach(f => f(d, ...msg))
-  Raven.captureException(msg, sendErr => {
-    if (sendErr) {
-      throttledConsoleError(
-        'Failed to send captured exception to Sentry',
-        sendErr.message
-      )
-    }
-  })
+  Sentry.captureException(msg)
```

Comment:
I take it this new SDK function is supposed to gracefully handle any errors that occur while trying to send the info?

Reply:
While I can't find any complete documentation of the API, I can't find any examples that use this parameter so I think it's been removed. I think the errors are logged to the console (though that's not a real help to us).